girder/web/core/tests/server.ts

```diff
@@ -43,7 +43,7 @@ export const setupServer = () => {
 
   test.beforeEach(async ({ page }) => {
     await startCoverage(page);
-    await page.goto(`/?apiRoot=${encodeURIComponent(`http://localhost:${port}/api/v1`)}`);
+    await page.goto(`http://localhost:${port}/`);
```

Comment:
This will allow us to remove the `index.html` within `core` altogether if we want, and instead build it as a library. I didn't do that yet since I wasn't sure how it would impact all the testing infrastructure.